EmployeeLib/Managed.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace EmployeeLib
+{
+    public class Managed: Employee
```

Comment:
class names should be noun not a verb, also managed is not an employee, so it should not have inherited from employee. maybe consider renaming the class

Reply:
Managed is an adjective here not a verb, and that was your suggestion in the first project

Reply:
Also, why Managed is not an Employee? Employee has FirstName, LastName, Salary and CalculatePerHourRate() 
a managed person may not have all of these?

Reply:
1- it was not a class it was interface, classes always needs to be nouns.
2- because if some one ask an employee what do they do, they say a manager or developer not managed. So managed is not an employee it can be applied to an employee.